doc/lspconfig.txt

```diff
@@ -382,14 +382,15 @@ Launches the requested (configured) client, but only if it successfully
 resolves a root directory. Note: Defaults to all configured servers matching
 the current buffer filetype.
 
-:LspStop [client_id] or [config_name]                              *:LspStop*
-Stops the server with the given client-id or config name. Defaults to stopping
-all servers active on the current buffer. To force stop a language server: >
-    :LspStop <client_id> ++force
-
-:LspRestart [client_id]                                          *:LspRestart*
-Restarts the client with the given client id, and attempts to reattach to all
-previously attached buffers.
+:LspStop {[client_id] or [config_name]} ...                        *:LspStop*
```

Comment:
lua syntax is not used for vim commands documentation. `{..}` means something different in vim docs.

```suggestion
:LspStop [client_id_or_config_name] ...                        *:LspStop*
```